src/components/sbb-action-group/sbb-action-group.stories.js

```diff
@@ -0,0 +1,264 @@
+import { h } from 'jsx-dom';
```

Comment:
Can you add the following story:
- a story which has set horizontal from to a breakpoint (e.g. medium). So we can track that with chromatic tests too.